### DISCUSSION TITLE
Xiisad - Chrome API & Extension Manifest | Saturday, 16 November 2024, 8:00 PM

### DISCUSSION THREAD

**Discussion by sharafdin** [dugsiiyeinc/dhaqsoPay#16] [Announcements]
Dhalinyaro diyaar garooba Insha'Allah!

**Comment by A-M-Ahmed** [comment]
insha allah bro.

**Comment by Nasrah-muse** [comment]
> Dhalinyaro diyaar garooba Insha'Allah!

**Comment by Nasrah-muse** [comment]
> Dhalinyaro diyaar garooba Insha'Allah!

weli lama bilaabin miyaa eng

**Reply by sharafdin** [reply]
waa 16-ka bisha ama sabtida

**Comment by hayle01** [comment]
ready eng
